examples/chart.rs

```diff
@@ -254,6 +254,7 @@ fn ui<B: Backend>(f: &mut Frame<B>, app: &App) {
                 .style(Style::default().fg(Color::Gray))
                 .bounds([0.0, 5.0])
                 .labels(vec!["0".bold(), "2.5".into(), "5".bold()]),
-        );
+        )
+        .legend_position(Some(LegendPosition::TopLeft));
```

Comment:
This doesn't show up on the example because charts are too smalls (at least on my screen), plus it shows the default position.

What do you think about doing:
- First chart stays the same
- Loosen `hidden_legend_constraints` so we can see the default legend on the second chart
-  Third chart, use non default position and loosen constraints to see it

Something like:
![image](https://github.com/ratatui-org/ratatui/assets/36198422/e90aad71-73c5-4f10-811b-1e88c9db3734)